### DISCUSSION TITLE
Feature Idea: Save/Copy/Paste Blocks

### DISCUSSION THREAD

**Discussion by jaschaio** [payloadcms/payload#4637] [Feature Requests & Ideas]
When editing a collection item I can duplicate blocks. Or I can duplicate collection items. But I can not "save" various versions of a given block. What if I want to reuse a certain block with certain values across multiple collection items? If I could just copy and paste a block from one collection item to another that would already remedy the issue. But as it stands now I always have to add the block and reconfigure all values unless they are set as default.

**Comment by a-axton** [comment]
This might not be the ideal setup but you might be able to create a new collection for "shared blocks" or something like that and then have a new block that has a relationship to that collection and can select the block you want

**Reply by jmikrut** [reply]
This is the move! And it’s already fully possible. We do it all the time.

**Comment by TorbenHammes** [comment]
This feature would be really helpful for us. We want to copy pages, articles, blogs etc.
